.travis.yml

```diff
@@ -16,27 +16,34 @@
 # specific language governing permissions and limitations
 # under the License.
 #
+sudo: true
```

Comment:
Is this added back for a specific reason? I thought sudo on xenail based infra was deprecated by Travis.

Reply:
Ah. yeah. Will remove. No need for that. Probably a rebase artifact.